clean_code_main/clean_code_checks/y_check_is_interface_in_class.clas.abap

```diff
@@ -30,7 +30,7 @@ CLASS Y_CHECK_IS_INTERFACE_IN_CLASS IMPLEMENTATION.
 
     settings-pseudo_comment = '"#EC INTF_IN_CLASS' ##NO_TEXT.
     settings-disable_threshold_selection = abap_true.
-    settings-threshold = 0.
+    settings-threshold = 1.
```

Comment:
Any impact on customers who are using this check already?

Reply:
no